src/elm/Main.elm

```diff
@@ -11,7 +12,7 @@ import Html as H exposing (Html)
 import Html.Attributes as A
 import Json.Decode as Decode exposing (Decoder)
 import Json.Decode.Pipeline as DecodeP
-import Notification exposing (Notification)
+import Notification exposing (Notification, decrementTimer)
```

Comment:
Please only expose types. It's much easier to see where things are when they are fully qualified. Types usually have a relation to their containing module (and there are fewer of them) so it's much easier to see where they are coming from.